### PR TITLE
[ETFE-3044] - Added page for change destination type (COD-02)

### PR DESCRIPTION
**Pull request by LewisMitchell**

### CODE REVIEW

File: app/navigation/InformationNavigator.scala [hunk 1]
```diff
@@ -38,12 +38,23 @@ class InformationNavigator @Inject()() extends BaseNavigator {
...
           case _ =>
             controllers.routes.DraftMovementController.onPageLoad(request.ern, request.arc)
         }
 
+    case ChangeDestinationTypePage =>
+      (userAnswers: UserAnswers) => userAnswers.get(ChangeDestinationTypePage) match {
+        case Some(wantsToChangeDestinationType) =>
+          if(wantsToChangeDestinationType) {
+            //TODO: redirect to COD-03
+            testOnly.controllers.routes.UnderConstructionController.onPageLoad()
```
Comment (MJCallahanPage) [inline app/navigation/InformationNavigator.scala:51]: The page already exists. It's the `DestinationTypeController`. Mike is going to refactor it to make sure it matches the CoD content and logic but 90% of it is probably already there.

File: test/navigation/InformationNavigatorSpec.scala [hunk 2]
```diff
@@ -69,6 +68,39 @@ class InformationNavigatorSpec extends SpecBase {
...
+
+        "must go to the New Destination Type page" - {
+
+          "when the user answers yes" in {
+            //TODO: redirect to COD-03
+            val userAnswers = emptyUserAnswers.set(ChangeDestinationTypePage, true)
+            navigator.nextPage(ChangeDestinationTypePage, NormalMode, userAnswers) mustBe
+              testOnly.controllers.routes.UnderConstructionController.onPageLoad()
+          }
+        }
```
Comment (MJCallahanPage) [inline test/navigation/InformationNavigatorSpec.scala:81]: GO to DesitnationTypeController

**Review by MJCallahanPage — CHANGES_REQUESTED** [review]
One small thing, otherwise LGTM :)

File: app/navigation/InformationNavigator.scala (hunk 1 above)
Comment (LewisMitchell) [reply app/navigation/InformationNavigator.scala:51]: Nice, will change

**Review by MJCallahanPage — APPROVED** [review]
LGTM 🟢 🚀 🧑‍🚀

**Pull request merged**